cstate/powermgr_cstate_tests.sh

```diff
@@ -0,0 +1,322 @@
+#!/bin/bash
+# SPDX-License-Identifier: GPL-2.0-only
+# Copyright (c) 2022 Intel Corporation
+# @Author   wendy.wang@intel.com
+# @Desc     Test script to verify Intel CPU Core Cstate functionality
+# @History  Created Nov 01 2022 - Created
+
+cd "$(dirname $0)" 2>/dev/null
+source ../.env
+source common.sh
```

Comment:
`source comon.sh` can be removed as it has been imported in **.env**

```shell
lkvs_root=$(realpath "$(dirname "${BASH_SOURCE[0]}")")

if [[ -z "$path_exported" ]]; then
  NEW_PATH="$PATH":$(find "$lkvs_root" -type d -not -path '*.git*' | tr "\n" ":")
  NEW_PATH=${NEW_PATH%:}
  export PATH="$NEW_PATH"
  source common.sh
  path_exported=1
fi
```

Reply:
Will remove, thanks!